src/lib/services/searchUtilActions/search.spec.ts

```diff
@@ -0,0 +1,210 @@
+import { expect } from '@jest/globals';
```

Comment:
As discussed yesterday, please rename to AasSearcher.spec.ts

Reply:
You are right, done